Packs/Cryptocurrency/Integrations/Cryptocurrency/Cryptocurrency.yml

```diff
@@ -0,0 +1,71 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Cryptocurrency
+  version: -1
+configuration:
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- defaultvalue: 7 days
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
+  name: first_fetch
+  required: false
+  type: 0
+- defaultvalue: '50'
+  name: max_fetch
+  required: false
+  type: 0
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
```

Comment:
by mistake right? we do not have fetch here.

Reply:
yes, for some reason its not the latest. we don't have any configuration for this integration. will update this now.